### ISSUE TITLE
Uncaught Error: Aligner: aligner-css has already been activated

### ISSUE THREAD

**Issue by DarrinTisdale** [adrianlee44/atom-aligner#34]
To reproduce:

1. Not sure -- it just happened a few minutes after upgrading all packages.

**Atom Version**: 1.0.18
**System**: Mac OS X 10.11
**Thrown From**: [aligner](https://github.com/adrianlee44/atom-aligner) package, v0.16.2

### Stack Trace

Uncaught Error: Aligner: aligner-css has already been activated

```
At /Volumes/Is/tizzyd/.atom/packages/aligner/lib/provider-manager.coffee:15

Error: Aligner: aligner-css has already been activated
  at Error (native)
  at ProviderManager.register (/Volumes/Is/tizzyd/.atom/packages/aligner/lib/provider-manager.coffee:15:20)
  at Aligner.registerProviders (/Volumes/Is/tizzyd/.atom/packages/aligner/lib/main.coffee:51:21)
  at Provider.module.exports.Provider.provide (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/provider.js:30:52)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/service-hub.js:55:26
  at doNTCallback0 (node.js:416:9)
  at process._tickCallback (node.js:345:13)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "unity-ui",
      "one-light-syntax"
    ],
    "disabledPackages": [
      "atom-fsharp",
      "atom-ternjs",
      "file-icon-supplement"
    ]
  }
}
```

### Installed Packages

```coffee
# User
Sublime-Style-Column-Selection, v1.3.0
aligner, v0.16.2
aligner-css, v1.2.0
aligner-php, v1.0.0
aligner-ruby, v1.4.0
aligner-scss, v1.1.0
atom-pair, v2.0.10
autoclose-html, v0.19.0
autocomplete-php, v0.3.7
autocomplete-plus-python-jedi, v0.3.6
hex, v0.6.2
highlight-line, v0.11.0
highlight-selected, v0.10.1
htmlhint, v1.1.3
language-fsharp, v0.8.4
linter-js-yaml, v1.1.2
linter-php, v1.0.3
linter-phpmd, v1.2.0
linter-scss-lint, v2.0.0
linter-tidy, v1.0.1
local-history, v3.2.3
wordpress-api, v1.2.1

# Dev
No dev packages
```

**Comment by Maelstrome26** [comment]
Can confirm.

**Comment by adrianlee44** [comment]
This should be fixed from `v0.17.0` forward. This was due to improper handling of enabling and disabling in aligner.

When you update aligner to v0.17.0, you will still see this bug. Restarting Atom will fix the problem and shouldn't appear in future updates.
